x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/create_field/required_parameters_forms/token_count_type.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+
+import { i18n } from '@kbn/i18n';
+
+import { AnalyzerParameter } from '../../../field_parameters';
+import { STANDARD } from '../../../../../constants';
+import { EditFieldSection } from '../../edit_field';
+
+export const TokenCountTypeRequiredParameters = () => {
+  return (
+    <EditFieldSection
+      title={i18n.translate(
+        'xpack.idxMgmt.mappingsEditor.tokenCountRequired.analyzer.sectionTitle',
+        {
+          defaultMessage: 'Analyzer',
+        }
+      )}
+    >
+      <AnalyzerParameter
+        path="analyzer"
+        label={i18n.translate(
```

Comment:
The label should come from the `parameter_definitions.ts` to avoid repeating the same translation, as "search_analyzer" does. And I just realized that we are missing `i18n` there 😊